From 60414152acefd1f34694e7fe06094d8cf2940cbe Mon Sep 17 00:00:00 2001
From: Leonard Crestez <leonard.crestez@nxp.com>
Date: Mon, 5 Mar 2018 14:34:55 +0200
Subject: [PATCH 0175/5242] MLK-11352-2 ARM: dts: imx7d-sdb: add dts support
 for pxp-v4l2-output

commit  24fdefc78d3e1c85140ba4425b7eb49d40ccaac0 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add dts support for pxp-v4l2-output module for
imx7d-sdb board.

Signed-off-by: Fancy Fang <chen.fang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-sdb.dts |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index 09e4166..30ba616 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -137,6 +137,11 @@
 		default-brightness-level = <6>;
 		status = "okay";
 	};
+
+	pxp_v4l2_out {
+		compatible = "fsl,imx7d-pxp-v4l2", "fsl,imx6sx-pxp-v4l2", "fsl,imx6sl-pxp-v4l2";
+		status = "okay";
+	};
 };
 
 &adc1 {
-- 
1.7.9.5

